cc.config.json

```diff
@@ -25,17 +23,8 @@
         "3d": {
             "modules": ["3d"],
             "dependentAssets": [
-                "1baf0fc9-befa-459c-8bdd-af1a450a0319",
                 "511d2633-09a7-4bdd-ac42-f778032124b3",
-                "5d45aa00-e064-4938-b314-4265f0c2258c",
-                "28d6d6b8-3f66-4a73-9795-17a0852ba2d4",
-                "ec8106fe-05bf-4e94-943c-e0d3b7bb5e45",
-                "620b6bf3-0369-4560-837f-2a2c00b73c26",
-                "2df0a40b-26c2-47ce-be0d-4d3cd4164737",
-                "39f74202-ead5-4a45-b966-273d526adbf1",
-                "8f6ac413-2f1e-4b88-b26f-54556b5dd510",
-                "45e7c0c8-2699-4912-b45f-d42bb8384189",
-                "84ac6f69-3086-455a-86a4-561da8ee710b"
+                "5d45aa00-e064-4938-b314-4265f0c2258c"
```

Comment:
"5d45aa00-e064-4938-b314-4265f0c2258c" is "deferred-lighting.effect", deferred pipeline need